src/main/java/eu/europeana/metis/sandbox/config/web/MvcConfig.java

```diff
@@ -31,7 +30,7 @@ public void addFormatters(FormatterRegistry registry) {
 
   @Override
   public void addCorsMappings(CorsRegistry registry) {
-    if (StringUtils.isNotBlank(corsMapping)) {
+    if (corsMapping.length != 0) {
```

Comment:
An empty array probably is allowed in `allowedOrigins` but a null array is not and we are not checking that here.
In fact `corsMapping.length` will fail if the field is `null` before the below `allowedOrigins` execution.
Better use `ArrayUtils.isNotEmpty(array)`

Reply:
Thanks for sharing this, this is definitely a better option, I will add it